### ISSUE TITLE
scan smartphone doesn't appears

### ISSUE THREAD

**Issue by openmotion** [Polidea/RxAndroidBle#670]
Hello
when i scan devices, my smartphones doesn't appears ( smartphone android 9 ), i have standard bluetooth devices (like my tv or printer)
when i scan in the bluetooth parameters of i have the others smartphones
press scan button in the sample app

**Comment by dariuszseweryn** [comment]
Let me rephrase your issue:

You scan devices using the sample app. In the app you can see your printer or your TV, you cannot see other smartphones though.

Correct?

If so then it is possible that your other smartphones do not advertise themselves. I cannot tell more as I have only what you have described ignoring the bug template.

**Comment by dariuszseweryn** [comment]
Closing due to lack of information and inactivity of the original poster.

When more data will be available—feel free to add that information and reopen this issue.

Best Regards
